operations/mimir/images.libsonnet

```diff
@@ -4,6 +4,9 @@
     memcached: 'memcached:1.6.17-alpine',
     memcachedExporter: 'prom/memcached-exporter:v0.6.0',
 
+    redis: 'redis:6.2-bullseye',
+    redis_exporter: 'oliver006/redis_exporter:latest',
```

Comment:
Definitely shouldn't be using `latest` here and I'd rather not pull in _any_ Docker image that isn't official. It's fine to leave is out for the first pass.

Reply:
This is "official" redis_exporter although not part of prometheus project. Yes I will peg the version and not use latest.